autovizwidget/autovizwidget/utils/configuration.py

```diff
@@ -0,0 +1,33 @@
+# Distributed under the terms of the Modified BSD License.
+from hdijupyterutils.constants import EVENTS_HANDLER_CLASS_NAME, LOGGING_CONFIG_CLASS_NAME
+from hdijupyterutils.utils import join_paths
+from hdijupyterutils.configuration import override as _override
```

Comment:
I don't like this API at all.  Don't make any changes for now, but let's rethink it and address it in a later PR :)